src/javascript/app/pages/trade/markets.jsx

```diff
@@ -80,9 +84,19 @@ class Markets extends React.Component {
             underlying_symbol = Object.keys(this.markets[market_symbol].submarkets[submarket].symbols).sort()[0];
         }
         const is_not_crypto = symbol => !/^(cry|JD)/i.test(symbol);
+        const is_synthetic = symbol => /^(synthetic)/i.test(symbol);
+        const is_uk = State.getResponse('authorize.country') === 'gb';
         const market_arr = Object.entries(this.markets).sort((a, b) => sortSubmarket(a[0], b[0]));
-        const non_crypto_markets_arr = market_arr.filter(market => is_not_crypto(market));
-        this.markets_all = non_crypto_markets_arr.slice();
+        final_markets_arr = market_arr.filter(market => is_not_crypto(market));
+        final_market_obj = Object.fromEntries(final_markets_arr);
+        if ((Client.isMalta() || is_uk) && Client.getAccountOfType('virtual')) {
+            final_markets_arr = market_arr.filter(market => is_synthetic(market));
+            final_market_obj = Object.fromEntries(final_markets_arr);
+            market_symbol = Object.keys(final_market_obj)[0];
+            const submarket = Object.keys(final_market_obj[market_symbol].submarkets).sort(sortSubmarket)[0];
+            underlying_symbol = Object.keys(final_market_obj[market_symbol].submarkets[submarket].symbols).sort()[0];
+        }
```

Comment:
How about we move lines 90, 91 into else block here to avoid extra `market_arr filter` when the client is from malta?

Reply:
```suggestion
        } else {
           final_markets_arr = market_arr.filter(market => is_not_crypto(market));
           final_market_obj = Object.fromEntries(final_markets_arr);
        }
```

Reply:
yeah , you're right, will change it